src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
 
+import { Movie } from '../../types/Movie';
+
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = movie;
```

Comment:
```suggestion
export const MovieCard: React.FC<Props> = (props) => {
  const { movie } = props;
  const {
    title,
    description,
    imgUrl,
    imdbUrl,
  } = movie;
```

Let's get used to such usage of destructuring with props